keyboards/xd75/keymaps/davidrambo/keymap.c

```diff
@@ -72,7 +72,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    NAVPC  , _______, _______, _______, _______, _______, _______, KC_MPLY, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
-   _______, KC_LCTL, KC_LGUI, _______, KC_LCTL, _______, _______, _______, _______, _______, _______, KC_RCTL, KC_RALT, KC_RGUI, _______
+   _______, KC_LCTL, KC_LGUI, _______, KC_LCTL, CBSPC  , _______, _______, _______, _______, _______, KC_RCTL, KC_RALT, KC_RGUI, _______
```

Comment:
Shouldn't this be: 
```suggestion
   _______, KC_LCTL, KC_LGUI, _______, KC_LCTL, C_BSPC, _______, _______, _______, _______, _______, KC_RCTL, KC_RALT, KC_RGUI, _______
```

Reply:
It's for a different macro.

https://github.com/qmk/qmk_firmware/blob/66533ee093d23f07ab93824ecd1dfcb9d763f681/keyboards/xd75/keymaps/davidrambo/keymap.c#L5 vs https://github.com/qmk/qmk_firmware/blob/66533ee093d23f07ab93824ecd1dfcb9d763f681/keyboards/xd75/keymaps/davidrambo/keymap.c#L12

Reply:
Ah, okay. Didn't check.

Well, that brings up another issue:  Those are pretty similarly named, and may be an issue for you, in the future. 


If you're okay with that, then by all means, and let us know.

Reply:
Hey, thanks for the comments. I changed the alias for "LCTL_T(KC_BSPC)" from "CBSPC" to "BS_CTL", which I think is an improvement in clarity. I echoed this change for the bottom layer's Mac OS X version, which now has the alias of "BS_GUI".

Reply:
Glad to have helped!  And looks good!